vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/init/ObserverMapperSettings.java

```diff
@@ -0,0 +1,156 @@
+ * <LI>instanceof my.mapper.class<br/>
+ * the parent ObserverSettings will be applied for all classes, which are instanceof my.mapper.class
+ * </UL>
+ * 
+ * @author Michael Remme
+ * 
+ */
+public class ObserverMapperSettings {
+  private static final String INSTANCEOF = "instanceof";
+  private String classDefinition;
+  private boolean instOf = false;
+  private Class<?> mapperClass;
+  private Class<? extends Annotation> annotation;
+
+  @SuppressWarnings("unused")
+  private ObserverMapperSettings() {
+    // only usable for serialization
+  }
+
+  /**
+   * Constructor for a new instance. The class definition can be something like "my.mapper.class" or "instanceof
+   * my.mapper.class"
+   * 
+   * @param classDefinition
+   */
+  public ObserverMapperSettings(String classDefinition) {
+    setClassDefinition(classDefinition);
+  }
+
+  /**
+   * Constructor for a new instance, which shall be executed on mapper classes, which are annotated with the given
+   * annotation
+   * 
+   * @param annotation
+   */
+  public ObserverMapperSettings(Class<? extends Annotation> annotation) {
+    setAnnotation(annotation);
+  }
+
+  private void init() {
+    if (classDefinition != null) {
+      String clsName = classDefinition;
+      int index = clsName.indexOf(INSTANCEOF);
+      if (index >= 0) {
+        instOf = true;
+        clsName = clsName.substring(index + INSTANCEOF.length()).trim();
+      }
+      try {
+        mapperClass = Class.forName(clsName);
+      } catch (ClassNotFoundException e) {
+        throw new MappingException(e);
+      }
+    }
+  }
+
+  /**
+   * The class definition which defines for which mapper classes the definition shall be applied
+   * 
+   * @return the classDefinition
+   */
+  public String getClassDefinition() {
+    return classDefinition;
+  }
+
+  /**
+   * The class definition which defines for which mapper classes the definition shall be applied
+   * 
+   * @param classDefinition
+   *          the classDefinition to set
+   */
+  public void setClassDefinition(String classDefinition) {
+    this.classDefinition = classDefinition;
+    init();
+  }
+
+  /**
+   * If an observer shall be executed for mapper classes, which are annotated with a certain Annotation, the annotation
+   * must be defined here
+   * 
+   * @return the annotation
+   */
+  public Class<? extends Annotation> getAnnotation() {
+    return annotation;
+  }
+
+  /**
+   * If an observer shall be executed for mapper classes, which are annotated with a certain Annotation, the annotation
+   * must be defined here
+   * 
+   * @param annotation
+   *          the annotation to set
+   */
+  public void setAnnotation(Class<? extends Annotation> annotation) {
+    this.annotation = annotation;
+  }
+
+  /**
+   * This method checks, whether the current definition is applyable to the given instance of IMapper.
+   * 
+   * @param mapper
+   * @return true, if appliable
+   */
+  boolean isApplyableFor(IMapper<?> mapper) {
```

Comment:
Could you do a Strg+H -> "applyable" and replace those as well? There are some variable and at least one more method in ObserverSettings